### ISSUE TITLE
Isn't this included in i18n now? missing redirect in README

### ISSUE THREAD

**Issue by MarkDBlackwell** [svenfuchs/i18n-missing_translations#3]
Thanks for all your contributions over the years to the highly useful i18n gem!

In a search for "i18n missing translation", this is the top Google hit.

For someone just starting in internationalization, it is non-obvious (IMHO) whether this is a module (currently) of gem i18n.

If so, whether it is included automatically (with gem i18n) also is non-obvious.

The installation steps are outdated, claiming that I18n 0.5.0 has not yet been released.

If this repository's functionality already is included in gem i18n:
- Every time Google leads people to research this repository, some of their time is wasted.
- Please consider adding a message at the top of this repository's GitHub README, to redirect them to gem i18n!

**Comment by jmuheim** [comment]
Hey, I faced the same issue. Searched for a way to track missing translations, got here, and no idea whether this is still valid information or not. Also, when I try to require it, Ruby doesn't find it.

**Comment by danielpuglisi** [comment]
+1

**Comment by dimitrypo** [comment]
+1 wasted an hour of my time

**Comment by rin** [comment]
that makes 5 hrs of wasted time now :)

**Comment by MarkDBlackwell** [comment]
@svenfuchs, @rwrede,  @jmuheim, @danielpuglisi, @Dvporg, @rin: perhaps pull request #6 will be accepted (or somehow help). :)

**Comment by svenfuchs** [comment]
Sorry for dragging my feet, everyone.

If anyone wants to take over maintenance for this repository please ping me. I am currently not using this any more and unfortunately I don't have time to maintain it.

You'd probably also want to be added as an owner on rubygems.org, so including your rubygems email address would be useful :)

**Comment by MarkDBlackwell** [comment]
> I am currently not using this any more and unfortunately I don't have time to maintain it.

Actually, is this exact repository (i18n-missing_translations) useful anymore? Should it be maintained, or instead, should it be declared historical?

Time has passed, but I seem to recall that this repository's basic functionality (finding missing translations) is now effectively included in your own [i18n library](https://github.com/svenfuchs/i18n). I.e., I seem to recall that I could find all missing translations easily using just the i18n library.

At least, would you be willing to check, and to confirm (or deny) this (transfer of effective functionality to i18n)? Or, alternatively, to delegate the checking—of whether this repository (i18n-missing_translations) is unnecessary—to another i18n library contributor)?

Now, I have added an [issue](https://github.com/svenfuchs/i18n/issues/246) for this over at library i18n.

Or do you actually mean you haven't time to work on your (own) i18n library anymore, and that you wish someone to take on the responsibility of the entire i18n library? Is this true? Just asking, in order to clarify. :)

Another simple, easy, and efficient alternative would be to accept my pull request, and then to revisit the issue if anyone complains—if they actually need this repository! :/ What do you think?

Obviously, easing internationalization is important. Many thanks again, for all your contributions to internationalization! :)

**Comment by svenfuchs** [comment]
@MarkDBlackwell no, it's not superseeded, and the functionality hasn't been moved to the i18n gem.

`i18n-missing_translations` adds logging of missing translations using the hook provided by `I18n`. It also adds a Rails hook, which probably is outdated/incompatible by now (not sure). This kind of functionality certainly wouldn't be included in the i18n gem directly which is supposed to be more generic.
